client/src/features/form/FormCreateEvent.js

```diff
@@ -1,14 +1,23 @@
 import { useContext } from "react";
+import { useState } from "react";
+import { useEffect } from "react";
 import { FormMoverContext } from "./contexts/FormMoverContext";
+import DataService from "services/dataService";
 
 export default function FormCreateEvent() {
   const { userData, setUserData } = useContext(FormMoverContext);
 
   const handleChange = e => {
     const { name, value } = e.target;
-    setUserData({ ...userData, [name]: value });
+    setUserData({ ...userData, [name]: value, discordName: user?.displayName });
   };
 
+  const [user, setUser] = useState();
```

Comment:
State ideally should be at top of the function I think. Should be moved to line 9.